renderer.js

```diff
@@ -44,6 +44,23 @@ function bindCanvas(c) {
   // renderer.gammaFactor = 2.2;
   renderer.shadowMap.enabled = true;
   renderer.shadowMap.type = THREE.PCFSoftShadowMap;
+
+  // reload renderer WebGLInfo.reset function
```

Comment:
What is being reloaded?

Reply:
Default renderer.info.reset method

Reply:
That's not really "reload". Reloading the page to collect stats is something very different, so this comment is confusing.